site/content/en/docs/Reference/fleet.md

```diff
@@ -174,3 +174,41 @@ The `spec` field is the actual `FleetAllocation` specification and it is compose
   when the `FleetAllocation` is created
 - `metadata` is an optional list of custom labels and/or annotations that will be used to patch 
   the game server's metadata in the moment of allocation. This can be used to tell the server necessary session data
+
+{{% feature publishVersion="0.8.0" %}}
+# Fleet Scale Subresource Specification
+
+Scale subresource is defined for a Fleet. Please refer to [Kubernetes docs](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#subresources).
+
+You can use the following command to scale the fleet with name simple-udp:
+```bash
+$ kubectl scale fleet simple-udp --replicas=10
+fleet.stable.agones.dev/simple-udp scaled
+```
+
+You can also use [Kubernetes API](../Guides/access-api.md) to get or update the Replicas count:
+```
+curl http://localhost:8001/apis/stable.agones.dev/v1alpha1/namespaces/default/fleets/simple-udp/scale
+...
+{
+  "kind": "Scale",
+  "apiVersion": "autoscaling/v1",
+  "metadata": {
+    "name": "simple-udp",
+    "namespace": "default",
+    "selfLink": "/apis/stable.agones.dev/v1alpha1/namespaces/default/fleets/simple-udp/scale",
+    "uid": "4dfaa310-2566-11e9-afd1-42010a8a0058",
+    "resourceVersion": "292652",
+    "creationTimestamp": "2019-01-31T14:41:33Z"
+  },
+  "spec": {
+    "replicas": 10
+  },
+  "status": {
+    "replicas": 10
+  }
+```
+
+Also exposing a scale subresource allows to configure HorizontalPodAutoscaler and PodDisruptionBudget for a fleet in the future.
```

Comment:
I wonder if we want to make a comment like "but this has not been tested, and is not currently supported - but if you are looking for this feature, please be sure to let us know in an [issue](https://github.com/GoogleCloudPlatform/agones/issues)" ?

Or something like that - just to set very clear expectations. WDYT?

Reply:
I was able to create HPA for a Fleet, so I think it seems possible to configure custom Stackdriver metric instead of default targetCPUUtilizationPercentage:
```
apiVersion: autoscaling/v1
kind: HorizontalPodAutoscaler
metadata:
  name: fleetHPA
  namespace: default
spec:
  scaleTargetRef:
    apiVersion: "stable.agones.dev/v1alpha1"
    kind: Fleet
    name: simple-udp
  minReplicas: 1
  maxReplicas: 10
  targetCPUUtilizationPercentage: 50
```